src/android/LocalNotification.java

```diff
@@ -90,6 +95,14 @@ public void initialize (CordovaInterface cordova, CordovaWebView webView) {
     public void onResume (boolean multitasking) {
         super.onResume(multitasking);
         deviceready();
+
+        Context context = cordova.getActivity().getApplicationContext();
+        AlarmManager alarmManager = (AlarmManager) context.getSystemService(Context.ALARM_SERVICE);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S && !alarmManager.canScheduleExactAlarms()) {
```

Comment:
```suggestion
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S && alarmManager.canScheduleExactAlarms()) {
```
?

Do we need the second condition?

Reply:
`!alarmManager.canScheduleExactAlarms()` checks whether schedule exact alarms is granted or not

Reply:
`alarmManager.canScheduleExactAlarms()` checks whether the app has the appropriate SCHEDULE_EXACT_ALARM permission or not